docs/src/manual.md

```diff
@@ -857,10 +857,23 @@ A common workflow used in BenchmarkTools is the following:
 
 1. Start a Julia session
 2. Execute a benchmark suite using an old version of your package
-3. Save the results somehow (e.g. in a JSON file)
+    ```julia
+    results = run(suite, verbose = true)
+    ```
+4. Save the results somehow (e.g. in a JSON file)
+    ```julia
+    BenchmarkTools.save("results.json", median(results))
```

Comment:
```suggestion
    BenchmarkTools.save("old_results.json", median(old_results))
```

Reply:
Why the median instead of the minimum?

Reply:
I don't know - there might be things where you can get lucky? Cache hit type of things. Or maybe you try to figure out if something is prime and your algorithm just tries to divide it by all numbers 2, 3, ... and it is lucky because it is divisible by two but that is not representative of the typical performance.

Median performance just seemed reasonable to me

Reply:
minimum is the default quantity in BenchmarkTools (the one showed by `@btime`) so I'm going with that one